.github/workflows/stale-management.yml

```
@@ -20,7 +20,7 @@ jobs:
       pull-requests: write
     steps:
       - name: Handle stale PRs and issues
-        uses: actions/stale@v9
+        uses: actions/stale@v8
```

actionlint (1.7.7)

the runner of "actions/stale@v8" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)